awx/main/dispatch/periodic.py

```diff
@@ -41,6 +42,7 @@ def run():
                         'encountered an error while scheduling periodic tasks'
                     )
                 time.sleep(idle_seconds)
+                GuidMiddleware.get_guid()
```

Comment:
Every time the periodic background scheduler wakes up, get a GUID so it's available in the current thread when `.apply_async()` is called to schedule work.